system/jlib/jstats.cpp

```diff
@@ -796,7 +826,7 @@ static const constexpr StatisticMeta statsMetaData[StMax] = {
     { SIZESTAT(GeneratedCpp) },
     { SIZESTAT(PeakMemory) },
     { SIZESTAT(MaxRowSize) },
-    { NUMSTAT(RowsProcessed) },
+    { NUMSTATAGR(RowsProcessed) },
```

Comment:
Not sure about this, or the way of indicating stats to aggregate - I'll come back to it later.